pages/landing/hipaa/index.html

```diff
@@ -52,6 +54,33 @@
     body: |
       The tools we all rely on (AWS, Terraform, Kubernetes, Docker), DevOps best practices, and auditor expectations are constantly changing. To help you remain HIPAA compliant, we offer commercial support and maintenance for our app and infrastructure code, all of which is semantically versioned, so you can stay completely up-to-date with the latest compliance requirements, security patches, and AWS/DevOps best practices via simple version number bumps.
 
+  - heading: Step 6. Verify all future code changes against Open Policy Agent (OPA) policies
+    precode: |
+      <figure><figcaption>Ensure all images come from a trusted registry</figcaption>
+           <pre>
+            <code class="language-python">
+              package kubernetes.admission
+
+              deny[msg] {
+                  input.request.kind.kind == "Pod"
+                  some i
+                  image := input.request.object.spec.containers[i].image
+                  not startswith(image, "hooli.com/")
```

Comment:
NIT: should we replace this with `gruntwork.io` (this rule is saying all kubernetes container images must source from `hooli.com`, which is an odd example for us to have on our website).

Reply:
Yup - fixed in 86869d52